gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started-kubernetes.adoc

```diff
@@ -0,0 +1,656 @@
++
+If you receive an `UPGRADE FAILED` response, see <<upgrade-failed,troubleshooting>> information about this issue. 
+
+* A new gateway is running in your terminal window.
+
+. Return to Runtime Manager: 
+.. Select *&#8592; Flex Gateways* from the side navigation panel of the Kubernetes *Add a Flex Gateway* page. 
++
+image::gateway-quickstart-k8-return-to-gateway-list.png["Return to Flex Gateway list in Runtime Manager"]
+.. Navigate to your registered gateway in the *Flex Gateways* page that opens:
++
+image::gateway-quickstart-k8-my-gateway-rtm.png["Flex Gateway my-gateway in Runtime Manager"]
++
+You can use the search field in the page to narrow the list of gateways.
+
+.. Make sure that the status of your gateway is *Connected*. 
++
+If the status is *Disconnected*, it is likely that the cluster configuration is incorrect or there is a network issue. Check <<troubleshooting>> for more information.
+//TODO: COVER BOTH TYPES OF ISSUES; MAKE SURE MOST COMMON ARE COVERED
+//      don't have network-related troubleshooting yet; 
+//      so have some config troubleshooting
+
+[[create-api]]
+== Step 5: Publish and Deploy a Simple API to Flex Gateway
+
+Create and deploy a simple API to your gateway in Anypoint API Manager. 
+
+. In Runtime Manager, click the *View APIs* menu option for your gateway to navigate to its *View APIs* page.
++
+image::gateway-quickstart-k8-view-apis.png["View APIs menu item for Flex Gateway"]
++
+Alternatively, you can navigate to API Manager from the Anypoint Platform menu.
+. From the page that opens, click *Add API* (and if present, *Add new API*) to navigate to the *APIs / Add API* page in API Manager.
+. From *APIs / Add API*, configure an API by working through the API configuration pages, starting with the *Runtime* configuration:
++
+image::gateway-quickstart-k8-select-gateway-for-api.png["Add API page with Flex Gateway Runtime and my-gateway selected"]
++
+.. From the *Runtime* configuration page, select *Flex Gateway* as your runtime.
+.. Find and select your gateway in the *Select a gateway* area of the page.
++
+Use the search field to locate it, if necessary.
+.. Click *Next* to open the *API* page.
+
+. In the *API* configuration page associated with *APIs / Add API*, provide an API name and asset type:
++
+image::gateway-quickstart-k8-api-config-api.png["Screenshot of 'API' page",width=60%]
++
+.. Click *Create new API*.
++
+The remaining procedures walk you through steps for creating and testing a new API that is running on your gateway. Notice, however, that it is also possible to select an existing API from Exchange from this page. That process is described in the xref:api-gateway::create-instance-task.adoc#add-a-new-api[Add a New API] within the API Manager documentation.
```

Comment:
I'm a bit unsure about what `a new API that is running on your gateway` means
```suggestion
The remaining procedures walk you through steps for creating and testing a new API in Exchange. Notice, however, that it is also possible to select an existing API from Exchange from this page. That process is described in the xref:api-gateway::create-instance-task.adoc#add-a-new-api[Add a New API] within the API Manager documentation.
```

Reply:
Is there a way to incorporate the gateway's role into the suggested change? Hoping to gently suggest why we're creating this API.

I'm not quite sure how to avoid an awkward sentence. Perhaps "...a new API in Anypoint Exchange for which the gateway routes traffic..." or something like that.

Reply:
yes